### PR TITLE
6353: Add apiProvider

### PR DESCRIPTION
**Pull request by kat1906**
AB#6353
AB#6384

### CODE REVIEW

**Comment by kat1906** [comment]
> Nice all I would add is a wrap around all the axios calls with a try catch and when we catch the error we log it and then just rethrow it again

yeah like I said this is just the stuff from the existing way as not had a chance to do the extra stuff yet

**Comment by kat1906** [comment]
depends whether you we want to put this in to make sure the concept is ok in the pipeline before adding extra stuff

**Comment by alibombaye** [comment]
Yeah that sounds fine to me. We can add a separate task to add the logging when we get an error. At least that way we should only have to make that change in the one place

File: src/apiProvider/index.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import axios from 'axios';
```
Comment (alibombaye) [inline src/apiProvider/index.js:1]: Just a comment on the location. Did we agree to put this is a `helpers` folder? or is there a reason it belongs better in the root?

**Pull request merged**